package/MDAnalysis/analysis/align.py

```diff
@@ -359,27 +415,141 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
     ref_coordinates = ref_atoms.positions - ref_com
     mobile_coordinates = mobile_atoms.positions - mobile_com
 
-    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates)
-
-    R, new_rmsd = rotation_matrix(mobile_coordinates, ref_coordinates, weights=weights)
+    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates, weights)
 
     if subselection is None:
-        atoms = mobile.universe.atoms
+        mobile_atoms = mobile.universe.atoms
     elif type(subselection) is str:
-        atoms = mobile.select_atoms(subselection)
+        mobile_atoms = mobile.select_atoms(subselection)
     else:
         try:
-            atoms = subselection.atoms
+            mobile_atoms = subselection.atoms
         except AttributeError:
             raise TypeError("subselection must be a selection string, a AtomGroup or Universe or None")
 
-    atoms.translate(-mobile_com)
-    atoms.rotate(R)
-    atoms.translate(ref_com)
-
+    #_fit_to DOES subtract center of mass, will provide proper min_rmsd
+    mobile_atoms, new_rmsd = _fit_to(mobile_coordinates,\
```

Comment:
I meant in reference to using brackets to fix the problem. I don't
understand that part, I recognize that the slash is a problem now.

On Tue, May 31, 2016 at 9:14 PM kain88-de <notifications@github.com> wrote:

> In package/MDAnalysis/analysis/align.py
> <https://github.com/MDAnalysis/mdanalysis/pull/856#discussion_r65299642>:
>
> >          except AttributeError:
> >              raise TypeError("subselection must be a selection string, a AtomGroup or Universe or None")
> >
> > -    atoms.translate(-mobile_com)
> > -    atoms.rotate(R)
> > -    atoms.translate(ref_com)
> > -
> > +    #_fit_to DOES subtract center of mass, will provide proper min_rmsd
> > +    mobile_atoms, new_rmsd = _fit_to(mobile_coordinates,\
>
> Look at the end of the line. A pep8 linter will also trllyou the exact
> problem.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/856/files/3dd1dff2df1b585040e7b346b2886e08741bac8f#r65299642>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKcARlIA0lGvDv8QbRbKaYhaNKt17dofks5qHQc8gaJpZM4Ig_Re>
> .
>
-- 
Have a wonderful day,
John J. Detlefs